pkg/stub/stub.go

```diff
@@ -808,6 +815,22 @@ func (stub *stub) StateChange(ctx context.Context, evt *api.StateChangeEvent) (*
 	return &api.StateChangeResponse{}, err
 }
 
+func (stub *stub) ValidateContainerAdjustment(ctx context.Context, req *api.ValidateContainerAdjustmentRequest) (*api.ValidateContainerAdjustmentResponse, error) {
+	handler := stub.handlers.ValidateContainerAdjustment
+	if handler == nil {
+		return &api.ValidateContainerAdjustmentResponse{}, nil
```

Comment:
In other stub methods we return `nil` when `handler == nil`. Why might we want to return a non-`nil` response in this case?

Reply:
Hmm, I don't think we return anywhere a nil response when the plugins side `handler == nil`. We always return an empty-valued response in those cases, just like here.

Reply:
/facepalm Nevermind, I'm not sure where I was looking when I saw this and I can't `grep` for it in the repo either, I guess I hadn't had enough coffee. Closing